jetty-core/jetty-http/src/test/java/org/eclipse/jetty/http/QuotedQualityCSVTest.java

```diff
@@ -114,23 +115,69 @@ public void testOWS()
     }
 
     @Test
-    public void testEmpty()
+    public void testEmptyListItems()
     {
         QuotedQualityCSV values = new QuotedQualityCSV();
-        values.addValue(",aaaa,  , bbbb ,,cccc,");
+        values.addValue(",aaaa,  , bbbb,\"\" ,,x;q=0,cccc,");
         assertThat(values, Matchers.contains(
             "aaaa",
             "bbbb",
+            "\"\"",
```

Comment:
We drop empty items, we don't preserve them.
They are useless anyway.

The spec says to ignore them too.
https://www.rfc-editor.org/rfc/rfc9110.html#section-5.6.1.2